client/src/pages/Prospects.js

```diff
@@ -61,6 +63,12 @@ const useStyles = makeStyles((theme) => ({
 const Prospects = () => {
   const classes = useStyles();
 
+  const gmailDialogShouldOpen = () => {
+    const qs = queryString.parse(window.location.search);
+    if(qs['gmail_dialog']) return true;
+    return false;
+  }
```

Comment:
instead of having it as a query string, should we wrap GmailDialog into a react route (that's not in a switch statement) at a higher order component? Then we can always make sure it shows up, and can show up pretty much anywhere in the app

Reply:
I hadn't though of doing it that way, that seems like a much better solution that using a query string. I'll try using a react route.